doc/source/reference/udf/mroonga_snippet_html.rst

```diff
@@ -20,7 +20,7 @@ highlighted keyword in context.
 Syntax
 ------
 
-``mroonga_snippet_html()`` has required parameter and optional parameter::
+``mroonga_snippet_html()`` has required parameter and optional parameter.
```

Comment:
Could you add signatures instead of replacing with `.`?

Reply:
What signatures mean?
Is it correct that below modification?

`.` -> `:`

Reply:
```rst
``mroonga_snippet_html()`` has required parameter and optional parameter::

  mroonga_snippet_html(document)
```

See also other documents in `doc/source/reference/udf/`.